usage-guides/decentralized-finance.md

```diff
@@ -1,107 +0,0 @@
-# Decentralized Finance
-
-To track and analyze your DeFi actions use the Decentralized Finance tab from the left side menu. You can choose from the different types of DeFi actions presented in the submenu.
-
-## Overview
-
-rotki provides an overview of your assets in the different Defi protocols.
-
-![Defi Overview](/images/defi_overview.png)
```

Comment:
We should remove this and any other screenshots used here if they are not used elsewhere